packages/lodestar/test/unit/sync/gossip/handler.test.ts

```diff
@@ -42,6 +43,7 @@ describe("gossip handler", function () {
     });
     const handler = new BeaconGossipHandler(chainStub, networkStub, dbStub, logger);
     await handler.start();
+    await sleep(100);
```

Comment:
We need to investigate this, putting random sleep in unit tests cannot be solution. That worked before?!

Reply:
it seems there is an issue in gossip or the way we handle gossip so I have to sleep(100) for unit tests in order to confirm the e2e tests, I'm investigating it.

Reply:
resolved the issue